src/openscm_runner/adapters/ciceroscm_adapter/make_scenario_files.py

```diff
@@ -0,0 +1,236 @@
+"""
+Module with functionality to make emission input files
```

Comment:
```suggestion
Module with functionality to make emission input files

TODO: optimise to speed up reading and writing
```

I'm not sure, but I'm guessing this could be sped up?